src/platform/user/authentication/components/account-transition/TransitionSuccessModal.jsx

```diff
@@ -31,19 +36,20 @@ export default function TransitionSucessModal({ visible, onClose }) {
       id="account-transition-success-modal"
       primaryButton={primaryButton}
       status="success"
-      title="You're using a Login.gov account"
+      title="Account transfer is complete"
     >
       <div className="container">
         <div className="row">
           <p>
-            Congratulations! You have sucessfully signed in with your newly
-            created <strong>Login.gov</strong> account. This account will give
-            you acess to the same benefits and services you would normally use
-            on VA.gov or My HealtheVet.
+            You have sucessfully transferred your My HealtheVet account to{' '}
+            <strong>{TRANSFERED_CSP}</strong>. Your credentials and information
+            are now associated with this new account. You have access to all of
+            the same benefits and services you would normally use on My
+            HealtheVet and VA.gov.
           </p>
           <p>
-            It is recommended that you continue to use this account as your
-            preferred credential.
+            As you continue to use VA services, please ensure that you use this
+            account to sign in.
           </p>
         </div>
       </div>
```

Comment:
What is the use case for the success modal vs the success page? Do we still need both?

Reply:
Success modal - After Transition is complete
Success page - (probably named incorrectly) is one of the tickets Joe asked for, IAL1 vs IAL2, the content on that page still needs to be vetted & complete.